docs/source/index.rst

```diff
@@ -74,18 +74,23 @@ Common
 
 - Start mavros and VRPN for mocap: ```roslaunch mslquad quad_vrpn.launch```
 
-- May need to change ```vrpn_server_ip``` in ```launch/vrpn.launch``` to the ip address of the mocap machine.
+- May need to change ```vrpn_server_ip``` in ```launch/vrpn.launch``` to the ip
+  address of the mocap machine.
 
 Trajectory Following
 ~~~~~~~~~~~~~~~~~~~~
 
-- A basic trajectory following controller is implemented in **src/px4_base_controller.cpp**.
+- A basic trajectory following controller is implemented in 
+  **src/px4_base_controller.cpp**.
 
     ```roslaunch mslquad default_controller.launch```
 
-- The controller accepts trajectory from topic ```command/trajectory```, which is of type *MultiDOFJointTrajectory*. It's important that the trajectory is updated at least 10Hz since the controller uses a simple lookahead strategy.
+- The controller accepts trajectory from topic ```command/trajectory```, which
+  is of type *MultiDOFJointTrajectory*. It's important that the trajectory is
+  updated at least 10Hz since the controller uses a simple lookahead strategy.
 
-- At the moment, the controller operates at a fixed height (only works for 2D trajectory). 3D trajectory is TODO. 
+- At the moment, the controller operates at a fixed height (only works for 2D
```

Comment:
this is no longer true, the controller works in 3D just fine

Reply:
Awesome! Removed.